plugin/api.go

```diff
@@ -314,6 +314,13 @@ type API interface {
 	// Minimum server version: 5.6
 	UploadFile(data []byte, channelId string, filename string) (*model.FileInfo, *model.AppError)
 
+	// OpenInteractiveDialog will open an interactive dialog on a user's client that
```

Comment:
If I don't know what interactive dialogs are this doesn't make much sense. I think this could use more details.